python-checks/src/main/java/org/sonar/python/PythonBuiltinFunctions.java

```diff
@@ -19,11 +19,22 @@
  */
 package org.sonar.python;
 
-import com.google.common.annotations.VisibleForTesting;
-import com.google.common.io.Resources;
+import java.io.BufferedReader;
+import java.io.ByteArrayOutputStream;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.io.ByteArrayOutputStream'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)